backend/apps/owasp/services/certificate_service.py

```
@@ -0,0 +1,61 @@
+            github_user=user,
+            tier=tier,
+            is_revoked=False,
+        ).exists()
+
+        if has_active_cert:
+            return
+
+        try:
+            provider = CertificateProviderFactory.get_provider()
+        except ValueError as e:
+            logger.exception("Failed to resolve certificate provider: %s", str(e))
```

Ruff (0.15.15)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)